contribs/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -50,7 +50,7 @@
       "name": "conductor.event-queues.sqs.enabled",
       "type": "java.lang.Boolean",
       "description": "Enable the use of AWS SQS implementation to provide queues for consuming events.",
-      "sourceType": "com.netflix.conductor.contribs.queue.sqs.config.SQSEventQueueConfiguration"
+      "sourceType": "com.netflix.conductor.sqs.config.SQSEventQueueConfiguration"
```

Comment:
Looks like the contribs module is being removed, but this file remains. Should this file go the new awssqs-queue module?

Reply:
ah yes, this fell off my to-do list, fixing this now